### ISSUE TITLE
Early stopping checks for CLI

### ISSUE THREAD

**Issue by ZachNagengast** [argmaxinc/WhisperKit#91]
Similar to the example app, we should bring over the options for early stopping and add CLI arguments for setting them.

https://github.com/argmaxinc/WhisperKit/blob/0f19f7ed484f56889bb383c6bd687018021764d8/Examples/WhisperAX/WhisperAX/Views/ContentView.swift#L1040-L1069

**Comment by atiorh** [comment]
Indeed, we will aim for parity between the WhisperAX sample app and the CLI before the stable release.
